### PR TITLE
[ecosystem] add mint event and initial supply to create token event

### PR DESCRIPTION
**Pull request by areshand**
## Motivation

add an initial balance to create token event, which will be consumed by downstream indexer
add a mint event as well. will add mint and burn later

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

cargo test -p framework passed

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/Token.move [hunk 1]
```diff
@@ -76,6 +76,13 @@ module AptosFramework::Token {
     struct CreateTokenEvent has drop, store {
         id: TokenId,
         token_data: TokenData,
+        initial_balance: u64,
+    }
+
+    /// mint token event. This event triggered when creator adds more supply to existing token
+    struct MintTokenEvent has drop, store {
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/Token.move:83]: Are you planning to emit the event in mint() in this PR or a follow up one?

Comment (areshand) [reply aptos-move/framework/aptos-framework/sources/Token.move:83]: there will be a follow up one for adding public facing mint function.  will emit the events there

Comment (movekevin) [reply aptos-move/framework/aptos-framework/sources/Token.move:83]: Makes sense 👍

**Review by movekevin — APPROVED** [review]

**Pull request merged**